main.go

```diff
@@ -3,12 +3,22 @@ package main
 import (
 	"log"
 
-	"github.com/raymundo/academy-go-q42021/router"
+	"github.com/rayl17/academy-go-q42021/controller"
+	parameters "github.com/rayl17/academy-go-q42021/global"
+	"github.com/rayl17/academy-go-q42021/router"
+	service_csv "github.com/rayl17/academy-go-q42021/service/csv"
+	"github.com/rayl17/academy-go-q42021/usecase"
 )
 
 func main() {
 
-	srv := router.NewRouter()
+	pokemonService, _ := service_csv.NewPokemonService(parameters.CsvPath)
+
+	pokemonUsecase := usecase.NewUseCase(pokemonService)
+
+	pokemonController := controller.NewController(pokemonUsecase)
+
+	srv := router.NewRouter(pokemonController)
```

Comment:
No need to leave too many empty lines. Also after line 13, remove the empty line. And also line 22 and 24.

```suggestion
	pokemonUsecase := usecase.NewUseCase(pokemonService)
	pokemonController := controller.NewController(pokemonUsecase)
	srv := router.NewRouter(pokemonController)
```

Consider in line 23 to use short syntax, as that's the only place where you are using `err`.